src/api/app/models/workflow/step/configure_repositories.rb

```diff
@@ -39,10 +43,25 @@ def target_project_base_name
   def validate_repositories
     return if step_instructions[:repositories].all? { |repository| repository.keys.sort == REQUIRED_REPOSITORY_KEYS }
 
+    # FIXME: This is only to help users migrate their configure_repositories steps when we introduced this breaking change. Remove this after March 1st, 2022.
```

Comment:
The date of _March 1st, 2022_ is really arbitrary. It gives users about a month to adapt their _configure_repositories_ step.